conans/client/cmd/new_ci.py

```diff
@@ -128,7 +126,6 @@
 gitlab = """
 variables:
     CONAN_USERNAME: "{user}"
-    CONAN_REFERENCE: "{name}/{version}"
```

Comment:
I'm not sure if we should remove this. It is not necessary if the recipe declares the name and version but necessary otherwise.

Reply:
correct, I almost forgot about that. taocpp does not use name and version in recipes, so conan reference is mandatory. I'll revert this line.